test/ERC4626Oracle.t.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+// types
+import {IERC20} from "forge-std/interfaces/IERC20.sol";
+import {IERC4626} from "forge-std/interfaces/IERC4626.sol";
+// libraries
+import {console2} from "forge-std/Test.sol";
+// contracts
+import {Test} from "forge-std/Test.sol";
+import {ERC4626Oracle} from "../src/erc4626/ERC4626Oracle.sol";
+
+contract ERC4626OracleTest is Test {
+    // erc4626 oracles
+    ERC4626Oracle yDaiOracle;
+    ERC4626Oracle yUsdcOracle;
+    ERC4626Oracle yWethOracle;
+
```

Comment:
Instead of using the Yearn v3 vaults, which don't seem to be live yet, you can find some live vaults [here](https://erc4626.info/vaults). I think that the Morpho-Comp and Morpho-AaveV2 vaults would do just fine.

The main problem is that if the exchange rate is 1000000000000000000 then we won't know if we are getting any code switched over. With an exchange rate different to that we can at least get some intuition, since shares tend to be more valuable than the underlting asset unless decimal shifts are involved.

Reply:
the vaults were updated shortly after and now have arbitrary share prices, which makes the tests effective. i used yearn mostly because they're very well-known.

let me know if there's still a need to switch over, more than happy to make the change